### PR TITLE
Add recording function

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,19 @@
-# Getting Started with Create React App
+# DrumMachine
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Capstone Project of the Web-Development bootcamp at neuefische
 
-## Available Scripts
+Drum Machine is an application that lets you play various samples on a Drum Rack or play on a Keyboard.
+In the Settings you can change the colors and the sample of your Drum Rack.
```
Comment (celinevo) [inline README.md:6]: Settings in lower case

File: README.md [hunk 2]
```diff
@@ -1,70 +1,19 @@
-# Getting Started with Create React App
+# DrumMachine
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Capstone Project of the Web-Development bootcamp at neuefische
 
-## Available Scripts
+Drum Machine is an application that lets you play various samples on a Drum Rack or play on a Keyboard.
```
Comment (celinevo) [inline README.md:5]: In your title `DrumMachine` is written without whitespaces, here with.

File: src/components/DrumLoopPlayer.js [hunk 3]
```diff
@@ -7,8 +7,12 @@ export default function DrumLoopPlayer({
 }) {
   return (
     <DrumLoopContainer>
-      <PlayPauseButton onClick={startDrumLoop} aria-label="play pause">
-        <img src={isPlayin} height="40px" width="40px" alt="play pause" />
+      <PlayPauseButton
+        type="button"
+        onClick={startDrumLoop}
+        aria-label="play pause"
```
Comment (celinevo) [inline src/components/DrumLoopPlayer.js:13]: Not wrong, but `pause play` sound a little smoother

**Review by celinevo — APPROVED** [review]
Everything looks good. I only found a couple spelling mistakes :)

File: src/components/DrumLoopPlayer.js [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
+        />
       </PlayPauseButton>
       <DrumLoopLabel htmlFor="drum-loop-select">
         Choose a Drum Loop
       </DrumLoopLabel>
       <DrumLoopSelect
         name="drum-loops"
         id="drum-loop-select"
-        onChange={getDrumLoop}
+        onChange={onChange}
```
Comment (teifler) [inline src/components/DrumLoopPlayer.js]: Really clean code and cool project! I just have one little recommendation change onChange to a different name to stop confusion.

File: src/components/DrumLoopPlayer.js [hunk 5]
```diff
@@ -31,6 +47,16 @@ export default function DrumLoopPlayer({
...
   );
+
+  function handleClick() {
+    startDrumLoop(isPlayin);
+    setIsPlayin(!isPlayin);
+  }
+  function onChange(event) {
+      const currentLoop = event.target.value;
+      getDrumLoop(isPlayin, currentLoop);
+      isPlayin ? setIsPlayin(false) : setIsPlayin(false);
```
Comment (ttaannjjaa) [inline src/components/DrumLoopPlayer.js:57]: Please check this here - i dont understand the condition having in both cases the same output

File: src/pages/DrumMachinePage.js [hunk 6]
```diff
@@ -93,31 +95,38 @@ export default function DrumMachinePage({ allPads }) {
...
-    loopPlayer.current.stop();
-    setIsPlayin(playbutton);
+  function getDrumLoop(isPlayin, currentLoop) {
+    if (recorder.current.state === 'inactive') {
+      loopPlayer.current.stop()
+      setCurrentDrumLoop(currentLoop);
+    } else {
+      recordStopClick();
+      setCurrentDrumLoop(currentLoop);
+
```
Comment (ttaannjjaa) [inline src/pages/DrumMachinePage.js]: Please check also here - setCurrentDrumLoop(currentLoop) is in both conditions - maybe you need to do it like this for coordination of drumloopplayer and record function, just to be sure :-)

**Review by ttaannjjaa — COMMENTED** [review]
Awesome work so far - congratulations for solving your problem - only 2 little comments for checking the code

**Pull request merged**